info_radius: 1.0
obstacle_cost: 70
filter_clear: 0.1